### DISCUSSION TITLE
Python Support for Column Headers?

### DISCUSSION THREAD

**Discussion by samwhealon** [aws-samples/amazon-textract-textractor#350] [Q&A]
I have been playing around with this library and the original textract-response-parser. I found that TRP doesn't support returning table_titles. So I looked at textractor, which supports titles but not column_headers.

Are there plans to add column headers to be created with the table object?

Column headers have a setter and return property under the table entity, however the column_headers setter doesn't seem to be accessed in the response parser. 

Where I think this would exist in the parser:
https://github.com/aws-samples/amazon-textract-textractor/blob/8cf3759adc4a9eee56f5ae1d15e778aa70bf88ca/textractor/parsers/response_parser.py#L1013

Functions for table exist here:
https://github.com/aws-samples/amazon-textract-textractor/blob/8cf3759adc4a9eee56f5ae1d15e778aa70bf88ca/textractor/entities/table.py#L152

Screenshot of my debugger. Column_headers register as true in the table_cells, but not populating in the column_headers field

![image](https://github.com/aws-samples/amazon-textract-textractor/assets/56609109/eeecc623-4974-4e16-8268-0fc8fa548b7a)

**Comment by Belval** [comment]
I upgraded this to an issue as I'd expect `column_headers` to provide the column headers.
